website/docs/how-to/how-to-schedule-feature-releases.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: How to schedule feature releases
+---
+import ApiRequest from '@site/src/components/ApiRequest'
+
+:::info Placeholders
+Placeholders in the code samples below are delimited by angle brackets (i.e. `<placeholder-name>`). You will need to replace them with the values that are correct in _your_ situation for the code samples to run properly.
+:::
+
+There's a whole host of reasons why you may want to schedule the release of a feature, such as:
+- **to release a feature at a specific date and time** (for a product launch, for instance)
+- **to make a feature available only up until a specific moment** (for a contest cutoff, for instance)
+- **to make a feature available during a limited period** (for a 24 hour flash sale, for instance)
+
+There's two distinct ways to do this, depending on which version of Unleash you are running:
+- If you're using version 4.9 or later of Unleash Pro or Enterprise, you can (and should) [use strategy constraints](#strategy-constraints)
+- Otherwise, [use custom activation strategies](#custom-activation-strategies)
+
+In this guide we'll schedule a feature for release at some point in time. The exact same logic applies if you want to make a feature available until some point in the future. Finally, if you want to only make a feature available during a limited time period, you can easily combine the two options.
+
+## Prerequisites
+
+This guide assumes that you've got the following:
+- some basic experience with Unleash
```

Comment:
Complete and utter nitpick but sometimes we use capitalised first letters in lists in this page and sometimes we don't, might be cleaner if we stick to one style unless there's overriding reasons to mix styles

Reply:
Totally agree that it would be cleaner! This is already merged in now, but I'll keep that in mind going forward. It _does_ depend a bit on the context (I think), though. E.g.:

I want to ...
- do a
- do b

Unleash has a number of security mechanisms built in:
- When blah blah ...
- The thingamabob ...


But maybe standardizing on always capitalizing would work well, now that I think about it 🤔  Great input either way!